examples/transmitter/swagger_server/events.py

```diff
@@ -29,18 +29,49 @@ class CAEPEvent(Event):
     initiating_entity: Optional[str]
     reason_admin: Optional[Dict[str, str]]
     reason_user: Optional[Dict[str, str]]
+    __uri__ = "https://schemas.openid.net/secevent/caep/event-type/"
 
 
 class SessionRevoked(CAEPEvent):
-    __uri__ = "https://schemas.openid.net/secevent/caep/event-type/session-revoked"
+    pass
+    
+    
+class TokenClaimsChange(CAEPEvent):
+    claims: Dict[str, str]
+
+
+class CredentialChange(CAEPEvent):
+    credential_type: str
+    change_type: str
+    friendly_name: Optional[str]
+    x509_issuer: Optional[str]
+    x509_serial: Optional[str]
+    fido2_aaguid: Optional[str]
+        
+
+class AssuranceLevelChange(CAEPEvent):
+    current_level: str
+    previous_level: str
+    change_direction: str
+
+    
+class DeviceComplianceChange(CAEPEvent):
+    current_status: str
+    previous_status: str
 
 
 class Events(BaseModel):
     # SSE
     verification: Optional[VerificationEvent] = Field(alias=VerificationEvent.__uri__)
 
     # CAEP
-    session_revoked: Optional[SessionRevoked] = Field(alias=SessionRevoked.__uri__)
+    CAEP_event: Optional[CAEPEvent] = Field(alias=CAEPEvent.__uri__)
+    session_revoked: Optional[SessionRevoked] = Field(alias=CAEPEvent.__uri__+"session-revoked")
```

Comment:
Instead of creating the URIs on the fly here, please create them in the model definitions like we do for `SessionRevoked` and `VerificationEvent`. I do like the way you are using a consistent URI as a base though. I think you could do
```
CAEP_BASE_URI = "https://schemas.openid.net/secevent/caep/event-type"

class SessionRevoked(CAEPEvent):
    __uri__ = f"{CAEP_BASE_URI}/session-revoked"
```
etc.